recipes/samtools/0.1.12/meta.yaml

```diff
@@ -0,0 +1,27 @@
+about:
+  home: https://github.com/samtools/samtools
+  license: MIT
+  summary: Tools for dealing with SAM and BAM files
+build:
+  number: 0
+package:
+  name: samtools
+  version: 0.1.12
+requirements:
+  build:
+    - gcc
+    - zlib
+    - ncurses
```

Comment:
pin to `5.8`

Reply:
@epruesse Shouldn't it be pinned to `{{CONDA_NCURSES}}` ?

Reply:
Err, yes, of course.